addons/account/populate/account_payment.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields
+from odoo.tools import populate
+
+from datetime import datetime, timedelta
+import logging
+import random
```

Comment:
You should'nt import random because populate give you a tool to get a seeded random object.
This is quite important to ensure generation determinism even when adding a new field/behaviour. 
Remove this import and fix the different call by getting one random object each time you need one ;)